docs/README.md

```diff
@@ -101,34 +101,40 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
 ![image](./images/user-guide/2-hipstershop.png)
 
 ## Run the load generator
+Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to simulate users traffic.  
 
-In another browser tab, navigate to the load-generator URL, from which you can simulate users interacting with the application to generate traffic. For this application, values like 100 total users with a spawn rate of 2 users per second are reasonable. Fill in the **Host** field with the "Hipster shop web address" from the installation stage if it isn't prepopulated. Click the **Start swarming** button to begin generating traffic to the site.
+- In another browser tab, navigate to the load-generator URL(from the installation stage if it isn't populated).  
```

Comment:
Nit: `load-generator URL(from` -> `load-generator URL (from`

Reply:
Changed it also in the website readme file